config/crowdtangle_list.json

```diff
@@ -1,4 +1,924 @@
 {
-	"crowdtangle_list_account_pairs": {},
-	"crowdtangle_saved_searches": {}
+	"crowdtangle_list_account_pairs": {
```

Comment:
Edit the content of this file to only show the following template. Not useful to show list names from the dashboard we are using.
{
	"crowdtangle_list_account_pairs": {},
	"crowdtangle_saved_searches": {}
}

Reply:
Should we add this to the .gitignore? It changes whenever you update the CTLists